src/contracts/engine/AmguConsumer.sol

```diff
@@ -23,7 +23,7 @@ contract AmguConsumer is DSMath {
         uint ethToPay = mul(
             sub(initialGas, gasleft()),
             mul(mlnPerAmgu, ethPerMln)
-        );
+        ) / 1 ether;
```

Comment:
@jennazenk Be aware of this one. Was talking to Simon and it looks like this is correct; the amgu payments we were getting before had 18 extra decimal places from what we expected

Reply:
thanks